qt/aqt/addcards.py

```diff
@@ -33,21 +33,16 @@
 )
 
 
-class AddCards(QMainWindow):
+class AddCards(QDialog):
```

Comment:
This is probably a problem (?)
I think I remember discussions about whether EditCurrent should also become a QMainWindow instead of a QDialog? Is the only difference the menu bar? However I don't see any other menu items in the AddCards menu bar.

Reply:
#980 describes why we changed it to a main window - it resulted in conflicting shortcuts. I presume we could make EditCurrent a main window in the future too, there just wasn't a pressing need or the change, as it didn't have field/notetype selectors.